README.md

```diff
@@ -7,48 +8,68 @@ The Ceeblue WebRTC Client is a generic client library designed to simplify the i
 - **Multi Bitrate Playback** - Enables the player to switch in real-time between multiple quality renditions based on network conditions and client capabilities.
 - **Adaptive Bitrate Streaming** - Allows the streamer to adapt its video bitrate in real-time based on network conditions and client capabilities.
 
+> [!CAUTION]
+>
+> Using TURN ensures near-universal connectivity, but introduces important tradeoffs:
+> - Relaying adds latency.
+> - TURN over TCP/TLS doesn’t handle congested networks well, video can freeze or slow down.
+> 
+> Use TURN wisely, it should be your last resort.
+
+
 ## Requirements
 
-| Item | Description |
-| --- | --- |
-| **Ceeblue Account** | To create a Stream, you will need an active account with [Ceeblue Streaming Cloud](https://dashboard.ceeblue.tv).<br>A trial account is sufficient. If you do not have one yet, you can request one on the [Ceeblue website](https://ceeblue.net/free-trial/). |
-| **Stream** | To use this library, you'll first need to create a stream either through [our Rest API](https://docs.ceeblue.net/reference) or on [the dashboard](https://dashboard.ceeblue.tv).<br><br>Use the following steps:<ol><li><a href="https://docs.ceeblue.net/reference/create-a-new-stream" target="_blank">Create a new stream</a></li><li>Copy the **Stream name (UUID)**</li><li>Copy the **Endpoint**</li></ol> |
-| **Node Package Manager (npm)** | Download and install from https://nodejs.org/en/download |
-| **http-server** | (Optional) Simple, zero-configuration command-line static HTTP server<br><br>The <a href="https://www.npmjs.com/package/http-server" target="_blank">http-server</a> is useful to explore the WebRTC client [examples](#examples) or the [documentation](#documentation) locally when you do not have a host.<br><br>To start the server, use the following command: `http-server . -p 8081`|
+#### 1. Node Package Manager (npm)
+Download and install npm from https://nodejs.org/en/download
+
+#### 2. Create a Ceeblue Account
+To create a Stream, you will need a Ceeblue account on [Ceeblue Cloud API] or [Ceeblue Dashboard]. A trial account is sufficient. If you do not have one yet, you can request one on the [Ceeblue website].
 
-### WebRTC URL format
+#### 3. Create a Stream
+To use this library you'll first need to create a stream either through [Ceeblue Cloud API] or via the [Ceeblue Dashboard]. Use the [Quick Start Guide] for fast results.
 
-**Stream name (UUID)** and **Endpoint** are part of the URL used to access the stream that will be given by Ceeblue either from [the API](https://docs.ceeblue.net/reference) or [the dashboard](https://dashboard.ceeblue.tv), and they are required to publish or play a stream. The WebRTC signalling URL format is as follows:
+Once the stream is created, retrieve its `<endpoint>` from the API in the `signallingUri` field. The value takes the following form, with an `in+` prefix when the stream is transcoded, or `as+` when it is not:
```

Comment:
I am not sure we need to explain the prefixes, this could change and we don't need to explain our internal cooking